renovate.json5

```diff
@@ -1,18 +1,35 @@
 {
+  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
   extends: ['github>netlify/renovate-config:default'],
   ignorePresets: [':prHourlyLimit2'],
   semanticCommits: true,
-  dependencyDashboard: true,
-  automerge: true,
+  // The config we're extending ignores test dirs, but we want to bump some fixture deps
+  ignorePaths: ['**/node_modules/**'],
   packageRules: [
+    // Since we've enabled Renovate (see above) for fixture sites, adjust the config for these.
     {
-      packageNames: [
-        // Those cannot be upgraded to requiring ES modules
-        'strip-ansi',
-      ],
-      major: {
-        enabled: false,
-      },
```

Comment:
Whatever this was, it's [already in our shared config](https://github.com/netlify/renovate-config/blob/36c644d32e2003632691d676d2ff739ca4204f4b/default.json#L254-L258)